docker-compose.yaml

```
@@ -0,0 +1,32 @@
+    networks:
+      - crawler-network
+
+volumes:
+  neo4j_data:
+  neo4j_logs:
+  neo4j_conf:
+  neo4j_plugins:
+
+networks:
+  crawler-network:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)